account_teams.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2020 Aiven, Helsinki, Finland. https://aiven.io/
```

Comment:
This one is missing `Update` (you can change team name). Probably not a big deal, as I would imagine that's not a very common use-case. On the API side, all resources are identified with IDs, and name fields are just for user convenience.

Reply:
Thanks, will add update method as well